### ISSUE TITLE
False positive superfluous blank line lint?

### ISSUE THREAD

**Issue by IndrajeetPatil** [r-lib/lintr#909]
If I have the following chunk in my `.Rmd` file,

````r
```{r child="man/rmd-fragments/doc.Rmd"}
```
````

`{styler}` will style it to

````r
```{r child="man/rmd-fragments/doc.Rmd"}

```
````

But this produces the following lint:

> Trailing blank lines are superfluous.

IMO, this is a false positive. Do you think the same?

**Comment by MichaelChirico** [comment]
can you confirm you're using the latest version? this may be fixed, I can't quite remember

**Comment by IndrajeetPatil** [comment]
Yes, I can confirm that I am using [devel version](https://github.com/IndrajeetPatil/statsExpressions/blob/7e62bd4369012d1a8506d4aec91c43c285359633/.github/workflows/lint.yaml#L25) of lintr. 

You can see the observed lints for yourself [here](https://github.com/IndrajeetPatil/statsExpressions/runs/5361243716?check_suite_focus=true).

**Comment by MichaelChirico** [comment]
ah, actually, I remember now. this is a big in styler IMO. otherwise it's related to #741

**Comment by IndrajeetPatil** [comment]
I had created related issue in _styler_ and my sense is that this (leaving a blank line in empty code chunk) is the desired behavior?

https://github.com/r-lib/styler/issues/918

**Comment by IndrajeetPatil** [comment]
Closing in favor of the original _styler_ issue.
